recipe/meta.yaml

```diff
@@ -351,6 +350,57 @@ outputs:
         - LICENSE.txt
       summary: C++ libraries for Apache Arrow Acero
 
+  - name: libarrow-compute
+    script: install-libarrow.sh  # [unix]
+    script: install-libarrow.bat  # [win]
+    version: {{ version }}
+    build:
+      string: h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ build_ext }}
+      run_exports:
+        - {{ pin_subpackage("libarrow-compute", max_pin="x.x") }}
+    requirements:
+      build:
+        - cmake <4
+        - ninja
+        # for strong run-exports
+        - {{ stdlib("c") }}
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        # needs to run protoc & grpc_cpp_plugin
+        - libgrpc                                # [build_platform != target_platform]
+        - libprotobuf                            # [build_platform != target_platform]
```

Comment:
are those required? they shouldn't :thinking:

Reply:
This was just C&P from some other output. 🙃